great_expectations/zep/bi_directional_dict.py

```diff
@@ -0,0 +1,62 @@
+import pprint
+from collections import UserDict
+from typing import Hashable, Iterable, Mapping, Optional, TypeVar
+
+T = TypeVar("T", bound=Hashable)
+
+
+class BiDict(
```

Comment:
we have a similar impl in `util.py`: https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/util.py#L109

I'm good keeping this one if you prefer

Reply:
Ah... yeah this looks familiar... probably where I got the idea 😆 .
I may keep this one due to the generics support. But I'll be sure and remove one or combine them once we leave the POC phase.